examples/index.md

```diff
@@ -162,3 +162,292 @@ Content-Type: application/vnd.api+json
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+If the API docs specified:
+
+> | Error Code | Title                                                                             |
+> |------------|-----------------------------------------------------------------------------------|
+> |  11        | "Invalid Attribute"                                                               |
+> |  123       | "too short"                                                                       |
+> |  124       | "emoji missing"                                                                   |
+> |  225       | "password must contain a letter, number, space, neologism, and special character" |
+> |  226       | "passwords do not match"                                                          |
+> |  227       | "password cannot be one of last five passwords"                                   |
+
+Multiple errors on `password` attribute, with error "code":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "code":   "123",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "225",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Frobnicate is not a neologism."
+    },
+    {
+      "code":   "226",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Password and password confirmation do not match."
+    }
+  ]
+}
+```
+
+Multiple errors on `first-name` attribute, with error "code", and "title":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "jsonapi": { "version": "1.0" },
+  "errors": [
+    {
+      "code":   "123",
+      "title":  "too short",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "124",
+      "title":  "emoji missing",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain an emoji"
+    }
+  ]
+}
+```
+
+Notice that this response includes not only the "errors" top-level member,
+but the "jsonapi" top-level member. It may also have include the "meta" or "links"
+top-level members, but not the "data" member.
+
+```http
+PATCH /posts/1
+
+{ "datum": { } }
+```
+
+```http
+HTTP/1.1 422 Bad Request
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Missing Data Member",
+      "source": { "source": "/data" },
+      "detail": "The required parameter, data, is missing."
+    }
+  ]
+}
+```
+
+Notice that the "source" indicates the error was causes not by a missing attribute, but
+by missing "data".
+
+
+### Responses to failed resource request:
+
+Basic request:
+
+```http
+GET /posts/1,2,3,4
```

Comment:
`/posts/1,2,3,4` is not a URI format that we recommend any more because (esp. with UUIDs) the URI length can somewhat quickly exceed the maximum length that many browsers/servers support. Personally, I think we should still be recommending this format for many cases (because it works, and offers a performance boost, 80% of the time). But, unless we add it back as a recommendation, I don't think we should have an example that uses it. 

Also, since `parameter` just refers to URI query parameters in the RFC 3986 sense, treating `/1,2,3,4` as a source of `"parameter"` errors is invalid.

There's probably a way to rework this example with a new URI to make it compliant, but maybe we should just get rid of it? After all, you already have a very good description of `"parameter"` in the next example.